pkgs/applications/editors/emacs/elisp-packages/manual-packages/elisp-ffi/default.nix

```diff
@@ -1,9 +1,10 @@
 {
   lib,
-  melpaBuild,
   fetchFromGitHub,
-  pkg-config,
   libffi,
+  melpaBuild,
+  pkg-config,
```

Comment:
Ideally, we should not move parameters around.

Reply:
It's nothing major, it could be put in a separate commit if it really matters but it's only three lines.

Reply:
Yeah, I also want to express that it is a nitpicking so I give my approval and use the word "ideally".

Reply:
Since I will be the maintainer of this package, let me keep this in my personal style.